src/clr-core/tag/tag.element.scss

```diff
@@ -42,14 +37,14 @@
     &:host(:hover),
     &:host(:active) {
       text-decoration: none;
-      background-color: var(--clr-tag-background-hover-color);
+      background-color: var(--background-hover);
       cursor: pointer;
     }
 
     &:host(:active) {
       // OldEdge/IE
-      box-shadow: 0 $clr_baselineRem_1px 0 0 var(--clr-tag-bg-color) inset;
-      box-shadow: 0 $clr-label-borderwidth 0 0 var(--clr-tag-bg-color) inset;
+      box-shadow: 0 $clr_baselineRem_1px 0 0 var(--background) inset;
+      box-shadow: 0 $clr-global-borderwidth 0 0 var(--background) inset;
```

Comment:
Why are we duplicating here? This looks like it was carried forward when it shouldn't have been.

Also `$clr-global-borderwidth` is not overrideable starting in 4.0. That may be fine in this particular instance.